src/main/java/com/github/vkorobkov/jfixtures/config/structure/tables/Tables.java

```diff
@@ -0,0 +1,46 @@
+package com.github.vkorobkov.jfixtures.config.structure.tables;
+
+import com.github.vkorobkov.jfixtures.config.structure.Section;
+import com.github.vkorobkov.jfixtures.config.structure.util.TableMatcher;
+import com.github.vkorobkov.jfixtures.config.yaml.Node;
+
+import java.util.stream.Stream;
+
+public class Tables extends Section {
+    private static final String SECTION_APPLIES_TO = "applies_to";
+    private static final String SECTION_PRIMARY_KEY = "pk";
+    private static final String SECTION_GENERATE = "generate";
+    private static final String SECTION_COLUMN = "column";
+    private static final String PK_DEFAULT_COLUMN_NAME = "id";
+
+    private final String name;
+
+    public Tables(Node node, String name) {
+        super(node);
+        this.name = name;
+    }
+
+    public boolean shouldAutoGeneratePk() {
+        return (boolean)getMatchingTables()
+                .map(node -> node.dig(SECTION_PRIMARY_KEY, SECTION_GENERATE).required())
```

Comment:
I would rename `SECTION_PRIMARY_KEY` into `SECTION_PK` and remove `SECTION_GENERATE` and `SECTION_COLUMN` at all since these are used only once and a string literal represents its content better than the constant